src/core/zowe/core_for_zowe_sdk/session.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, props: dict[str, Any]) -> None:
             self.session.user = props.get("user")
             self.session.password = props.get("password")
             self.session.reject_unauthorized = bool(props.get("rejectUnauthorized"))
+            self.session.response_timeout = props.get("responseTimeout")
```

Comment:
@zFernand0 @traeok Is it ok to store `response_timeout` on the `Session` class since it's specific to z/OSMF? Wondering if we should keep the properties on the `Session` class more profile/service-agnostic.

Reply:
Thank you for the review, can you review the changes I tried moving it to the zosmf package :relaxed: